vet.sh

```diff
@@ -120,11 +114,26 @@ if [[ -z "${VET_SKIP_PROTO}" ]]; then
     (git status; git --no-pager diff; exit 1)
 fi
 
-# - Check that our modules are tidy.
+# - gofmt, goimports, golint (with exceptions for generated code), go vet,
+# go mod tidy.
+# Perform these checks on each module inside gRPC.
 if go help mod >& /dev/null; then
-  find . -name 'go.mod' | xargs -IXXX bash -c 'cd $(dirname XXX); go mod tidy'
-  git status --porcelain 2>&1 | fail_on_output || \
+  for MOD_FILE in $(find . -name 'go.mod'); do
+    MOD_DIR=$(dirname $MOD_FILE)
+    if [[ $MOD_DIR == "./test/tools" ]]; then
+      continue
+    fi
+    pushd $MOD_DIR
+    go vet -all ./... | fail_on_output
+    gofmt -s -d -l . 2>&1 | fail_on_output
+    goimports -l . 2>&1 | not grep -vE "\.pb\.go"
+    golint ./... 2>&1 | not grep -vE "/testv3\.pb\.go:"
+
+    go mod tidy
+    git status --porcelain 2>&1 | fail_on_output || \
     (git status; git --no-pager diff; exit 1)
```

Comment:
Maybe it's just the way the diffs are presented, but if not: please add a couple spaces to this line so it's obvious it's a continuation of the command on the line above.